tutor/resources/styles/components/top-nav-bar.less

```diff
@@ -21,49 +52,45 @@
     }
   }
 
-  background: @tutor-white;
-  z-index: @zindex-navbar-fixed;
-  position: fixed;
-  top: 0;
-  right: 0;
-  left: 0;
-  z-index: @zindex-navbar-fixed;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  height: @tutor-navbar-height;
-
-
-  padding-right: @navbar-padding-horizontal;
-  padding-left:  @navbar-padding-horizontal;
-
   a {
-    color: #777;
+    color: inherit;
     text-decoration: none;
+
+    &:hover,
+    &[aria-expanded="true"] {
+      color: #333;
+    }
   }
 
   > * {
     display: flex;
-    align-items: center;
   }
+
   .ui-brand-logo {
-    margin-top: 3px; // the text on the logo is a bit vertically mis-aligned
     display: inline-block;
-    height: 41px;
-    width: 265px;
+    margin-top: 12px;
+    height: 34px;
+    width: 200px;
     background-repeat: no-repeat;
```

Comment:
The Tutor logo is just a bit high vertically.  I think that's supposed to be vertically centered?
![screen shot 2017-07-31 at 3 43 39 pm](https://user-images.githubusercontent.com/79566/28797471-0b36008a-7607-11e7-8c71-cdc5d7bf4e0b.png)
Rather than rely on the margin to center vertically, you could remove the `margin-top` from `.ui-brand-logo`, and set `.left-side-controls` to flex, and `.brand`  to ` display: flex; align-items: center;`